### DISCUSSION TITLE
Date format?

### DISCUSSION THREAD

**Discussion by mzupan** [laurens94/vue-timeline-chart#20] [Q&A] [answered]
I'm wondering if there's anyway to force it to do daily blocks and format the time format in the header? 

RIght now with code i'm seeing this

<img width="1208" alt="Screenshot 2024-12-29 at 4 29 37 PM" src="https://github.com/user-attachments/assets/dae91f31-18c9-4e70-a18d-bd5c24c2e3f9" />

I want to see each day for the next 2 weeks if possible 

thanks

EDIT

ok figured out the formatting like

```
:renderTimestampLabel="(timestamp) => {
      const date = new Date(timestamp)
      return `${(date.getMonth() + 1).toString()}/${date.getDate().toString()}`
    }"
```

That still just shows 1/1 and 1/8

**Comment by laurens94** [comment]
The logic for determining which timestamps to render is not customizable yet. 
I'll try to make this configurable in the next release!

Thanks for bringing this to my attention! 👍

**Comment by laurens94** [comment, marked as answer]
Actually there's a simpler workaround. You can decrease the `minTimestampWidth` prop so that the component allows more timestamps in the current view. 

It now renders per-week because the amount of days times the default minTimestampWidth (100px) won't fit in the viewport.

Let me now if this works!

**Comment by mzupan** [comment]
@laurens94 thanks.. that seems to be a good option.. i only give 3 choices to change the timerange so i can just hard code that setting for each time range to get that to work..

thanks!

**Reply by laurens94** [reply]
Nice! I also added an option to overwrite the default time unit snapping options, using the `scale` prop:

Information:
https://laurens94.github.io/vue-timeline-chart/guide/timestamps-and-scale.html

Example:
https://laurens94.github.io/vue-timeline-chart/examples/custom-timestamp-intervals.html
